packages/graph/hash_graph/lib/graph/src/store/postgres/context/links.rs

```diff
@@ -103,6 +106,8 @@ pub async fn read_links_by_target(
             FROM links
             JOIN type_ids ON version_id = link_type_version_id
             WHERE target_entity_id = $1
+            -- When reading links by target, the ordering becomes irrelevant, as the target could
+            -- have a variety of link that are meaningless to sort by their order.
```

Comment:
What about when getting links by their source? Are we ordering those?

Reply:
We are, yes. 
https://github.com/hashintel/hash/blob/b9e2860afa017a61077b89232d1bfcbebc5ce160/packages/graph/hash_graph/lib/graph/src/store/postgres/context/links.rs#L64
and
https://github.com/hashintel/hash/blob/b9e2860afa017a61077b89232d1bfcbebc5ce160/packages/graph/hash_graph/lib/graph/src/store/postgres/context/links.rs#L87

The comment was sort of a drive-by as it looked out of place that there was no ordering on this query

Reply:
Oh I didn't realise this was already implemented. Great 👍